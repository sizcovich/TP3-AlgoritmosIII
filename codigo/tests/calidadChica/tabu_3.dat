13 15 14 1033300
13 15 14 801200
5 25 24 4141900
16 35 34 6126500
11 45 44 9845200
9 55 54 16830800
16 65 64 11951800
36 75 74 12259600
14 85 84 21399600
46 95 94 32560700
12 105 104 46940600
