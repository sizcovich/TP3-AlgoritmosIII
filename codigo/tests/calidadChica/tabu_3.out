13 1 5 
13 1 13 
5 2 1 3 
16 2 1 32 
11 1 15 
9 1 18 
16 2 1 18 
36 1 15 
14 1 37 
46 1 24 
12 1 5 
